modules/sdk-core/src/bitgo/inscription/iInscription.ts

```diff
@@ -0,0 +1,15 @@
+import { PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+
+export interface IInscription {
+  prepareReveal(inscriptionData: Buffer, contentType: string, feeRate: number): Promise<PreparedInscriptionRevealData>;
```

Comment:
Should the UI use `FileReader.readAsArrayBuffer()` to get file data and pass it here as `inscriptionData`?
Ref: https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsArrayBuffer

Reply:
Not sure if that exact method but something along those lines yes, read the file as a buffer.